src/api-v1.js

```diff
@@ -740,26 +740,64 @@ api.declare({
   route: '/state/:workerType',
   name: 'state',
   title: 'Get AWS State for a worker type',
-  scopes: [['aws-provisioner:view-worker-type:<workerType>']],
+  output: 'state-response.json#',
```

Comment:
what about `/abridged-state/:workerType`, `/state-summary/:workerType` or `/capacity-info/:workerType`?  it's not ec2 objects being returned by that method, rather an internal format that's quite different to the ec2 objects.  In my mind, the 'real' call is the authenticated one and this one should only be used by those lacking the credentials to load the auth'd call.